include/wx/localedefs.h

```diff
@@ -108,6 +108,22 @@ struct WXDLLIMPEXP_BASE wxLanguageInfo
     const char* TrySetLocale() const;
 };
 
+// ----------------------------------------------------------------------------
+// wxLocaleIdent: contains platform dependent language name
+// ----------------------------------------------------------------------------
+
+class wxLocaleIdent
+{
+public:
+    wxLocaleIdent() = default;            // Leave name empty
```

Comment:
Unfortunately we still can't use C++11 in wx
```suggestion
    wxLocaleIdent() { } // Leave name empty
```